### PR TITLE
New SQL script to migration license agreement from FAS2 to FAS3

### PR DESCRIPTION
**Pull request by pypingou**
This is a start we likely need to discuss a little more some of the changes here,including:

-  do we port the old CLAs ?
- do we delete the old groups as proposed here?

But at least this is a start and should get us going :)

### CODE REVIEW

File: fas2_to_fas3_cla_groups.sql [hunk 1]
```diff
@@ -0,0 +1,72 @@
+-- Insert a new license agreement we will port the old one to
+
+INSERT INTO license_agreement (
+    "name", status, content, comment, enabled_at_signup,
+    creation_timestamp, update_timestamp)
+VALUES (
+    'cla_fpca',
```
Comment (laxathom) [inline fas2_to_fas3_cla_groups.sql]: Since we have a dedicated table management for license's agreement, can we get rid of `cla_` prefix?

**Comment by laxathom** [comment]
I'd say, port old CLA and archive them as well as related groups.

**Comment by pypingou** [comment]
Well, in FAS2 there isn't a relation CLA <-> Groups, it's either you signed one of the CLAs and that gave you access to all the groups or you had signed none and you were blocked.

Also, we got the green light for dropping all the old CLAs but the FPCA, so I think we're good to go :)

**Comment by ryanlerch** [comment]
Closing this Pull Request as the FAS project is now archived, not actively developed, and unmaintained.

FAS was replaced in March 2021 by Fedora Accounts (https://accounts.fedoraproject.org).

If this PR is a Feature Addition that might be beneficial to Fedora Accounts, consider reworking and refiling it against [Noggin](https://github.com/fedora-infra/noggin)

**Pull request closed without merging**